playbooks/tasks/portal-docker-services-start.yml

```diff
@@ -137,6 +137,10 @@
     docker_containers_check_list: "{{ docker_compose_services }}"
     docker_containers_check_timeout_secs: "{{ fifteen_minutes_in_secs }}"
 
+# Include running any commands post startup
+- name: Include running any commands post startup
+  include_tasks: tasks/portal-execute-commands.yml
```

Comment:
I would move it one task below, because logging status to `started` is directly related to docker compose above